ambry-frontend/src/test/java/com.github.ambry.frontend/AmbryBlobStorageServiceTest.java

```diff
@@ -1216,62 +1285,48 @@ private void doPostGetHeadUpdateDeleteTest(Account toPostAccount, Container toPo
 
   /**
    * Tests blob conditional TTL udpate and DELETE operations on the given {@code container}.
```

Comment:
typo: update

Reply:
Fixed ! (also fixed a similar typo in another class)